src/ray/raylet/scheduling/cluster_task_manager_test.cc

```diff
@@ -570,6 +573,48 @@ TEST_F(ClusterTaskManagerTest, OwnerDeadTest) {
   ASSERT_EQ(pool_.workers.size(), 1);
 }
 
+TEST_F(ClusterTaskManagerTest, TestInfeasibleTaskWarning) {
```

Comment:
Since infeasible tasks have their own separate queue, can we check that their backlogs/counts are correctly reported in the heartbeats? (You can probably just add those checks to the existing test cases.

Reply:
This seems to be already added.